tallycash/extension
# Add Polygon to Selectable Networks

This PR adds Polygon to the list of selectable networks in our top menu protocol selector.

### To Test
- Set MULTI_NETWORK=true in your .env file.
- Compile & load the app.
- The network dropdown menu should contain polygon as an options.

**Note that selecting the network does not actually do anything yet**

<img width="386" alt="image" src="https://user-images.githubusercontent.com/94649004/168605399-781f4704-7016-4ca1-91bd-5b5d5b17d858.png">
